src/App.js

```diff
@@ -1,23 +1,89 @@
-import React from 'react';
+/* eslint-disable no-unused-vars */
 import './App.css';
+import React, { Component } from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+  useRouteMatch,
+  useParams,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+function App() {
+  const styles = [
+    {
+      display: 'flex',
+      justifyContent: 'space-around',
+    },
 
-  render() {
-    return (
-      <div className="App">
-        <div className="appContainer">
-          <h1>Welcome, Math Magicians!</h1>
-          <Calculator />
+    {
+      margin: '20px',
+    },
+
+    {
+      listStyle: 'none', display: 'flex',
+    },
+
+    {
+      textDecoration: 'none', fontFamily: 'Franklin Gothic Medium', color: 'green',
+    },
+    {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+    },
+
+    {
+      fontFamily: 'Franklin Gothic Medium',
+    },
+  ];
+  return (
+    <>
+      <Router>
+        <div style={styles[4]}>
+          <div style={styles[0]}>
+            <h1 style={styles[5]}> Math Magicians</h1>
+            <nav>
+              <ul style={styles[2]}>
+                <li style={styles[1]}>
+                  <Link style={styles[3]} to="/">Home</Link>
+                </li>
+                <li style={styles[1]}>
+                  <Link style={styles[3]} to="/calculator">Calculator</Link>
+                </li>
+                <li style={styles[1]}>
+                  <Link style={styles[3]} to="/quotes">Quotes</Link>
```

Comment:
* Kindly fix the header section as the heading should be on the right and nav links to the left [Refer to the project general layout](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md#:~:text=The%20general%20layout%20should%20match%20these%20wireframes%3A)

Reply:
Done